archinfo/arch.py

```diff
@@ -305,6 +311,13 @@ def library_search_path(self, pedantic=False):
             path = sum([[x + 'tls/${ARCH}/', x + 'tls/', x + '${ARCH}/', x] for x in path], [])
         return map(subfunc, path)
 
+    vex_support = True
+    unicorn_support = True
+    address_types = (int, long)
+    function_address_types = (int, long)
+    supported_register_types = { 'SimSymbolicMemory', 'SimFastMemory' }
+    supported_memory_types = { 'SimSymbolicMemory', 'SimFastMemory', 'SimAbstractMemory' }
```

Comment:
We can't name angr objects from archinfo...

Reply:
Yeah... Otherwise this information will have to be hardcoded somewhere in `angr`, which is what I wanted to avoid. Let me come up with a better solution.